### PR TITLE
Refactor shield drawing code to be more sane and default plain shields to 2px rounded corners

### PR DESCRIPTION
**Pull request by ZeLonewolf**
This is an initial refactor pass to enable fixes for the current shield banner shadow overlap, which is impossible to fix currently due to code complexity.  I'm hoping this edit reduces the amount of complexity in the shield drawing code and makes it easier for contributors to work with.

In addition, I refactored out commonality in the shield definition for "default" white rectangles.  They had a mix of 1px and 2px rounded corners, so I defaulted this to 2px shields, which I think looks a little nicer.

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -1431,7 +1414,7 @@ export function loadShields(shieldImages) {
     },
   };
 
-  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX"] = shields["default"];
   shields["US:TX:Andrews:Andrews:Loop"] = banneredShield(
     roundedRectShield("white", "#003f87", "#003f87", 1, 1, null),
```
Comment (claysmalley) [inline style/js/shield_defs.js:1419]: Let's change this one to a 2px radius too. Same for the rest of the rectangles defined for Texas, Wyoming and Nova Scotia.

Comment (ZeLonewolf) [reply style/js/shield_defs.js]: Fixed in 6f5db25

File: style/js/shield.js [hunk 2]
```diff
@@ -242,9 +237,14 @@ function generateShieldCtx(id) {
     return ShieldDraw.blank();
   }
 
+  // Handle special case for Kentucky
```
Comment (claysmalley) [inline style/js/shield.js:240]: Kentucky is the only place that's taking advantage of this right now, but certainly other route networks will need this special casing too.

Comment (ZeLonewolf) [reply style/js/shield.js:240]: That's terrifying.

Comment (claysmalley) [reply style/js/shield.js:240]: Off the top of my head, HCTRA and some New York State Parkways are gonna need this treatment.

**Review by claysmalley — APPROVED** [review]
Looks good! Thanks for doing this.

**Pull request merged**